llama_index/schema.py

```diff
@@ -618,7 +637,26 @@ class ImageDocument(Document):
 
     # base64 encoded image str
     image: Optional[str] = None
+    image_path: Optional[str] = None
+    image_url: Optional[str] = None
 
     @classmethod
     def class_name(cls) -> str:
         return "ImageDocument"
+
+    def resolve_image(self) -> Union[str, bytes]:
```

Comment:
nit do we need to repeat those logic for both ImageDocument and ImageNode

Reply:
Good point, I can fix that

Reply:
Fixed